src/app/shared/support/support.ts

```diff
@@ -0,0 +1,16 @@
+import {Component, NgModule} from '@angular/core';
+
+@Component({
+  selector: 'app-support',
+  templateUrl: './support.html',
+  styleUrls: ['./support.scss']
+})
+export class Support {
+}
+
+@NgModule({
+  exports: [Support],
+  declarations: [Support],
+})
+export class SupportModule {
```

Comment:
We probably don't need another `NgModule`; this could just be another component in the homepage module